installing/installing_aws/installing-aws-outposts-remote-workers.adoc

```diff
@@ -61,6 +61,14 @@ include::modules/installation-applying-aws-security-groups.adoc[leveloffset=+2]
 
 include::modules/installation-aws-editing-manifests.adoc[leveloffset=+1]
 
+// Customizing Cluster Network MTU
+include::modules/installation-aws-cluster-network-mtu.adoc[leveloffset=+2]
```

Comment:
@dfitzmau Will you remove all Outpost contents from this PR?

Reply:
Well spotted. Removed.